migrations/V187__Add_System_Intake_Doc_Types.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TYPE system_intake_document_type RENAME VALUE 'DRAFT_ICGE' TO 'DRAFT_IGCE';
+ALTER TYPE system_intake_document_type ADD VALUE 'ACQUISITION_PLAN_OR_STRATEGY' BEFORE 'DRAFT_IGCE';
+ALTER TYPE system_intake_document_type ADD VALUE 'REQUEST_FOR_ADDITIONAL_FUNDING' AFTER 'DRAFT_IGCE';
+ALTER TYPE system_intake_document_type ADD VALUE 'MEETING_MINUTES' AFTER 'REQUEST_FOR_ADDITIONAL_FUNDING';
+
+CREATE TYPE system_intake_document_version AS ENUM('CURRENT', 'HISTORICAL');
+
+ALTER TABLE system_intake_documents ADD COLUMN document_version system_intake_document_version;
+
+UPDATE system_intake_documents SET document_version='CURRENT' WHERE document_version IS NULL;
+
+ALTER TABLE system_intake_documents ALTER COLUMN document_version SET NOT NULL;
```

Comment:
should we add a default here?

Reply:
Good question!

I prefer the approach that's here (using an `UPDATE` followed by a `SET NOT NULL`) as it allows us to set the `document_version` for all _existing_ documents, but still requires it as an input for all future documents (instead of assuming one value or another arbitrarily)

Since `document_version` should be a required input in GQL (and is required in the UI) I like having no default, and a `NOT NULL` constraint!

Reply:
I considered adding a default, but I thought we would want postgres to throw an error if we were not actively setting an option. I'd rather we were explicit in the SQL we write.

Reply:
makes sense!